docs/_includes/components/styled-list.html

```diff
@@ -0,0 +1,26 @@
+<div class="nhsnotify-list">
+  {% assign i = 1 %}
+  {% for item in include.data %}
+  <div id="write-your-messages" class="nhsuk-grid-row nhsnotify-list__item">
```

Comment:
IDs _should_ be unique across a whole page so best to only hard code them on very high level wrappers where we can be confident that there will only be a single one.  Can we remove it or pass it in if we really need it?

Reply:
Thanks, I missed this ID when turning it into a component. I've removed the ID, it's not needed IMO.